### ISSUE TITLE
Error Installing on Mac 

### ISSUE THREAD

**Issue by MrBobot** [JuliaImages/ImageView.jl#182]
Using Julia 1.2 and Mac 10.14.6 get the following problem:

```
(v1.2) pkg> add ImageView
 Resolving package versions...
 Installed RoundingIntegers ─ v0.2.0
 Installed Reactive ───────── v0.8.3
 Installed GtkReactive ────── v0.6.0
 Installed ImageView ──────── v0.9.0
 Installed Gtk ────────────── v0.18.0
  Updating `~/.julia/environments/v1.2/Project.toml`
  [86fae568] + ImageView v0.9.0
  Updating `~/.julia/environments/v1.2/Manifest.toml`
  [4c0ca9eb] + Gtk v0.18.0
  [27996c0f] + GtkReactive v0.6.0
  [86fae568] + ImageView v0.9.0
  [a223df75] + Reactive v0.8.3
  [d5f540fe] + RoundingIntegers v0.2.0
  Building Gtk → `~/.julia/packages/Gtk/mVhmL/deps/build.log`
┌ Error: Error building `Gtk`: 
│ Error: glib: undefined method `uses_from_macos' for Formulary::FormulaNamespace625eb1438427cb13d7fd495d335349a0::Glib:Class
│ ERROR: LoadError: ArgumentError: `brew info` failed for ["cairo", "gdk-pixbuf", "glib", "libcroco", "pango"]!
```

I tried to manually add ```Gtk```, but that leads to errors about ```Cairo```, and trying to solve that leads to more errors, and then tried using ```BinDeps``` to solve Cairo problems but that leads to further error in using  BinDeps.debug("Cairo") ... I'm stuck.

**Comment by johnnychen94** [comment]
This is duplicate to https://github.com/JuliaImages/ImageView.jl/issues/175

In simple words, `ImageView.jl` is broken on MacOS because the upstream package `Cairo.jl` doesn't work well.

**Comment by timholy** [comment]
Should be fixed if you use Julia 1.3 and up-to-date packages
